YIF.Core.Domain/ApiModels/ResponseApiModels/InstitutionOfEducationResponseApiModel.cs

```diff
@@ -98,6 +98,12 @@ public class InstitutionOfEducationResponseApiModel
         /// <example>true</example>
         public bool IsFavorite { get; set; }
 
+        /// <summary>
+        /// Is the institutionOfEducation banned
+        /// </summary>
+        /// <example>true</example>
+        public bool IsBanned { get; set; }
+
```

Comment:
the field is added to the wrong model
this field should be in InstitutionsOfEducationResponseApiModel, not in InstitutionOfEducationResponseApiModel

Reply:
fixed